assets/js/stripe.js

```diff
@@ -762,7 +690,7 @@ jQuery( function( $ ) {
 		 * Removes all Stripe errors and hidden fields with IDs from the form.
 		 */
 		reset: function() {
-			$( '.wc-stripe-error, .stripe-source, .stripe-intent' ).remove();
+			$( '.wc-stripe-error, .stripe-source' ).remove();
```

Comment:
Consider adding a comment as to why the .stripe-intent field is no longer removed?

Reply:
I only introduced the field in `sca/master`, so technically it was _never_ removed.

Reply:
I mean, it is used below with the others. Why isn't it still a good idea to remove it here?

Reply:
I see why this is confusing: I ~am~ was reusing the class in `render_payment_intent_inputs`. 

The intent field that was being removed in `reset` was a hidden field, generated by JavaScript, holding the intent after `handleCardPayment`. Since that field only exists within `sca/*` branches, adding a comment why it was removed when it was never there, might only end up being more confusing.

Reply:
I understand now - in addEventListener we have #stripe-intent-id, #stripe-intent-return and #stripe-intent-error and those are NOT the same as the other selectors used elsewhere  i.e. .wc-stripe-error and .stripe-source

Withdrawing my concern